### PR TITLE
solution

### PR DESCRIPTION
**Pull request by devWizHub**
[DEMO LINK] https://devWizHub.github.io/react_keyboard/

### CODE REVIEW

**Review by agniya-i — CHANGES_REQUESTED** [review]
Please deploy your changes, demo link isn't working now

**Comment by devWizHub** [comment]
I don't know why, but the demo link is not working after re-deployment. Any ideas what went wrong?

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+      {/* Keep this input for tests */}
+      <input
+        type="text"
+        value={inputValue}
+        onChange={handleChange}
+      />
+
+      <p className="App__message">
+        {key === '' ? 'Nothing was pressed yet'
+          : `The last pressed key is [${key}]`}
```
Comment (Anastasiia-Svintsova) [inline src/App.tsx:35]:
```suggestion
        {!key
          ? 'Nothing was pressed yet'
          : `The last pressed key is [${key}]`}
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Hey! Did you run npm run deploy? Are there any errors? Feel free to ask questions in the chat or visit Q&A sessions

**Pull request closed without merging**